### ISSUE TITLE
Missing texture for unwatered Rich Soil Farmland

### ISSUE THREAD

**Issue by mj717** [vectorwing/FarmersDelight#274]
Part of the texture(?) of unwatered Rich Soil Farmland seems to be missing. The sides of the block just show up as the pink and black checkered boxes. As soon as there is a crop planted and the item is watered, the sides of the block show up. 

I've attached a picture below of the un-hoed Rich Soil, Rich Soil Farmland before it's gained water, and Rich Soil Farmland with crops and water.

<img width="1440" alt="Screen Shot 2021-05-31 at 8 07 05 PM" src="https://user-images.githubusercontent.com/84801197/120254566-c0560580-c24f-11eb-919a-d1184a7de8d3.png">

**Comment by vectorwing** [comment]
You are using a resource pack, from the looks of it. Could you disable it to see if the issue persists? Also, check your game logs for errors when loading the resource pack in; missing textures are always reported.

Chances are the resource pack deals with farmland defaults, and might be breaking blocks that depend on it. Either this, or it had Rich Soil Farmland integration in the past.

**Comment by mj717** [comment]
Yes, I was using the resource pack Clarity (no OptiFine/OptiFabric). Removing Clarity did remove the issue and the block texture was as it should be. I'm sorry for not thinking of testing that before reporting this as an issue!

**Comment by vectorwing** [comment]
It's okay! Do post your logs if you can, anyway, as it could indicate a problem on my end as well.

**Comment by vectorwing** [comment]
I downloaded the Clarity pack to check out which files it edits, and it seems to directly change `template_farmland` as I suspected. It changes it to use a `#side` texture key, which Rich Soil Farmland does not have, causing it to break.

Perhaps I can add this extra key to the Rich Farmland model, but it might break things. I'll see how it works out, though perhaps the pack author should not be editing templates like that.
